sdk/translation/azure-ai-translation-text/setup.py

```diff
@@ -1,75 +1,72 @@
-#!/usr/bin/env python
-
-#-------------------------------------------------------------------------
+# coding=utf-8
+# --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#--------------------------------------------------------------------------
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+# coding: utf-8
 
-from io import open
 import os
-from setuptools import setup, find_packages
 import re
+from setuptools import setup, find_packages
 
 
 PACKAGE_NAME = "azure-ai-translation-text"
-PACKAGE_PPRINT_NAME = "Text Translation"
+PACKAGE_PPRINT_NAME = "Azure Ai Translation Text"
```

Comment:
I am not sure about this change, please confirm that it is expected change,

Reply:
It was changed by the code generation so I assume it is expected.